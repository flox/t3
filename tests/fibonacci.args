-- bash -c 'n=100; x=0; y=1; z=$x; function out() { if [ `expr $1 % 2` -eq 0 ]; then echo "$1: evens to stdout"; else echo "$1: odds to stderr" 1>&2; fi }; for i in `seq 1 $n`; do x=`expr $x + $y`; y=$z; z=$x; out $x; sleep 0.01; done'
