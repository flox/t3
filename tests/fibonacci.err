1: odds to stderr
1: odds to stderr
3: odds to stderr
5: odds to stderr
13: odds to stderr
21: odds to stderr
55: odds to stderr
89: odds to stderr
233: odds to stderr
377: odds to stderr
987: odds to stderr
1597: odds to stderr
4181: odds to stderr
6765: odds to stderr
17711: odds to stderr
28657: odds to stderr
75025: odds to stderr
121393: odds to stderr
317811: odds to stderr
514229: odds to stderr
1346269: odds to stderr
2178309: odds to stderr
5702887: odds to stderr
9227465: odds to stderr
24157817: odds to stderr
39088169: odds to stderr
102334155: odds to stderr
165580141: odds to stderr
433494437: odds to stderr
701408733: odds to stderr
1836311903: odds to stderr
2971215073: odds to stderr
7778742049: odds to stderr
12586269025: odds to stderr
32951280099: odds to stderr
53316291173: odds to stderr
139583862445: odds to stderr
225851433717: odds to stderr
591286729879: odds to stderr
956722026041: odds to stderr
2504730781961: odds to stderr
4052739537881: odds to stderr
10610209857723: odds to stderr
17167680177565: odds to stderr
44945570212853: odds to stderr
72723460248141: odds to stderr
190392490709135: odds to stderr
308061521170129: odds to stderr
806515533049393: odds to stderr
1304969544928657: odds to stderr
3416454622906707: odds to stderr
5527939700884757: odds to stderr
14472334024676221: odds to stderr
23416728348467685: odds to stderr
61305790721611591: odds to stderr
99194853094755497: odds to stderr
259695496911122585: odds to stderr
420196140727489673: odds to stderr
1100087778366101931: odds to stderr
1779979416004714189: odds to stderr
4660046610375530309: odds to stderr
7540113804746346429: odds to stderr
19740274219868223167: odds to stderr
31940434634990099905: odds to stderr
83621143489848422977: odds to stderr
135301852344706746049: odds to stderr
354224848179261915075: odds to stderr
