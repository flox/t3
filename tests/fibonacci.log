[36m[0m[1m[33m1: odds to stderr[0m
[36m[0m[1m[33m1: odds to stderr[0m
[36m[0m2: evens to stdout[0m
[36m[0m[1m[33m3: odds to stderr[0m
[36m[0m[1m[33m5: odds to stderr[0m
[36m[0m8: evens to stdout[0m
[36m[0m[1m[33m13: odds to stderr[0m
[36m[0m[1m[33m21: odds to stderr[0m
[36m[0m34: evens to stdout[0m
[36m[0m[1m[33m55: odds to stderr[0m
[36m[0m[1m[33m89: odds to stderr[0m
[36m[0m144: evens to stdout[0m
[36m[0m[1m[33m233: odds to stderr[0m
[36m[0m[1m[33m377: odds to stderr[0m
[36m[0m610: evens to stdout[0m
[36m[0m[1m[33m987: odds to stderr[0m
[36m[0m[1m[33m1597: odds to stderr[0m
[36m[0m2584: evens to stdout[0m
[36m[0m[1m[33m4181: odds to stderr[0m
[36m[0m[1m[33m6765: odds to stderr[0m
[36m[0m10946: evens to stdout[0m
[36m[0m[1m[33m17711: odds to stderr[0m
[36m[0m[1m[33m28657: odds to stderr[0m
[36m[0m46368: evens to stdout[0m
[36m[0m[1m[33m75025: odds to stderr[0m
[36m[0m[1m[33m121393: odds to stderr[0m
[36m[0m196418: evens to stdout[0m
[36m[0m[1m[33m317811: odds to stderr[0m
[36m[0m[1m[33m514229: odds to stderr[0m
[36m[0m832040: evens to stdout[0m
[36m[0m[1m[33m1346269: odds to stderr[0m
[36m[0m[1m[33m2178309: odds to stderr[0m
[36m[0m3524578: evens to stdout[0m
[36m[0m[1m[33m5702887: odds to stderr[0m
[36m[0m[1m[33m9227465: odds to stderr[0m
[36m[0m14930352: evens to stdout[0m
[36m[0m[1m[33m24157817: odds to stderr[0m
[36m[0m[1m[33m39088169: odds to stderr[0m
[36m[0m63245986: evens to stdout[0m
[36m[0m[1m[33m102334155: odds to stderr[0m
[36m[0m[1m[33m165580141: odds to stderr[0m
[36m[0m267914296: evens to stdout[0m
[36m[0m[1m[33m433494437: odds to stderr[0m
[36m[0m[1m[33m701408733: odds to stderr[0m
[36m[0m1134903170: evens to stdout[0m
[36m[0m[1m[33m1836311903: odds to stderr[0m
[36m[0m[1m[33m2971215073: odds to stderr[0m
[36m[0m4807526976: evens to stdout[0m
[36m[0m[1m[33m7778742049: odds to stderr[0m
[36m[0m[1m[33m12586269025: odds to stderr[0m
[36m[0m20365011074: evens to stdout[0m
[36m[0m[1m[33m32951280099: odds to stderr[0m
[36m[0m[1m[33m53316291173: odds to stderr[0m
[36m[0m86267571272: evens to stdout[0m
[36m[0m[1m[33m139583862445: odds to stderr[0m
[36m[0m[1m[33m225851433717: odds to stderr[0m
[36m[0m365435296162: evens to stdout[0m
[36m[0m[1m[33m591286729879: odds to stderr[0m
[36m[0m[1m[33m956722026041: odds to stderr[0m
[36m[0m1548008755920: evens to stdout[0m
[36m[0m[1m[33m2504730781961: odds to stderr[0m
[36m[0m[1m[33m4052739537881: odds to stderr[0m
[36m[0m6557470319842: evens to stdout[0m
[36m[0m[1m[33m10610209857723: odds to stderr[0m
[36m[0m[1m[33m17167680177565: odds to stderr[0m
[36m[0m27777890035288: evens to stdout[0m
[36m[0m[1m[33m44945570212853: odds to stderr[0m
[36m[0m[1m[33m72723460248141: odds to stderr[0m
[36m[0m117669030460994: evens to stdout[0m
[36m[0m[1m[33m190392490709135: odds to stderr[0m
[36m[0m[1m[33m308061521170129: odds to stderr[0m
[36m[0m498454011879264: evens to stdout[0m
[36m[0m[1m[33m806515533049393: odds to stderr[0m
[36m[0m[1m[33m1304969544928657: odds to stderr[0m
[36m[0m2111485077978050: evens to stdout[0m
[36m[0m[1m[33m3416454622906707: odds to stderr[0m
[36m[0m[1m[33m5527939700884757: odds to stderr[0m
[36m[0m8944394323791464: evens to stdout[0m
[36m[0m[1m[33m14472334024676221: odds to stderr[0m
[36m[0m[1m[33m23416728348467685: odds to stderr[0m
[36m[0m37889062373143906: evens to stdout[0m
[36m[0m[1m[33m61305790721611591: odds to stderr[0m
[36m[0m[1m[33m99194853094755497: odds to stderr[0m
[36m[0m160500643816367088: evens to stdout[0m
[36m[0m[1m[33m259695496911122585: odds to stderr[0m
[36m[0m[1m[33m420196140727489673: odds to stderr[0m
[36m[0m679891637638612258: evens to stdout[0m
[36m[0m[1m[33m1100087778366101931: odds to stderr[0m
[36m[0m[1m[33m1779979416004714189: odds to stderr[0m
[36m[0m2880067194370816120: evens to stdout[0m
[36m[0m[1m[33m4660046610375530309: odds to stderr[0m
[36m[0m[1m[33m7540113804746346429: odds to stderr[0m
[36m[0m12200160415121876738: evens to stdout[0m
[36m[0m[1m[33m19740274219868223167: odds to stderr[0m
[36m[0m[1m[33m31940434634990099905: odds to stderr[0m
[36m[0m51680708854858323072: evens to stdout[0m
[36m[0m[1m[33m83621143489848422977: odds to stderr[0m
[36m[0m[1m[33m135301852344706746049: odds to stderr[0m
[36m[0m218922995834555169026: evens to stdout[0m
[36m[0m[1m[33m354224848179261915075: odds to stderr[0m
