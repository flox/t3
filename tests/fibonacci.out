2: evens to stdout
8: evens to stdout
34: evens to stdout
144: evens to stdout
610: evens to stdout
2584: evens to stdout
10946: evens to stdout
46368: evens to stdout
196418: evens to stdout
832040: evens to stdout
3524578: evens to stdout
14930352: evens to stdout
63245986: evens to stdout
267914296: evens to stdout
1134903170: evens to stdout
4807526976: evens to stdout
20365011074: evens to stdout
86267571272: evens to stdout
365435296162: evens to stdout
1548008755920: evens to stdout
6557470319842: evens to stdout
27777890035288: evens to stdout
117669030460994: evens to stdout
498454011879264: evens to stdout
2111485077978050: evens to stdout
8944394323791464: evens to stdout
37889062373143906: evens to stdout
160500643816367088: evens to stdout
679891637638612258: evens to stdout
2880067194370816120: evens to stdout
12200160415121876738: evens to stdout
51680708854858323072: evens to stdout
218922995834555169026: evens to stdout
